examples/feature_selection/plot_feature_selection_pipeline.py

```diff
@@ -36,5 +37,7 @@
 y_pred = anova_svm.predict(X_test)
 print(classification_report(y_test, y_pred))
 
-coef = anova_svm[:-1].inverse_transform(anova_svm['linearsvc'].coef_)
```

Comment:
Is it not still useful to demonstrate this inverse_transform? Do we show it elsewhere?